cfssl.tf

```diff
@@ -107,32 +108,32 @@ data "ignition_file" "cfssl_iptables_rules" {
 -A INPUT -p icmp -m icmp -s "${var.cluster_subnet}" --icmp-type 11 -j ACCEPT
 COMMIT
 EOS
-  }
+
+}
```

Comment:
Looks like the upgrade tool has swallowed the indentation and added an extra space here.